specification/network/resource-manager/Microsoft.Network/stable/2022-05-01/webapplicationfirewall.json

```diff
@@ -689,12 +689,27 @@
           "description": "The state of the managed rule. Defaults to Disabled if not specified.",
           "type": "string",
           "enum": [
-            "Disabled"
+            "Disabled",
+            "Enabled"
           ],
           "x-ms-enum": {
             "name": "ManagedRuleEnabledState",
             "modelAsString": true
           }
+        },
+        "action": {
```

Comment:
Hey Sindhu, any reason we are deviating from what AFD policy has? I see AFD has ActionType field.
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-04-01/webapplicationfirewall.json

Reply:
Put/Get json would still be the same as the variable name is "action" for AFD too. Will change the type from Action to ActionType to be in sync with AFD.

Reply:
While your current change will work, can we not have it like in AFD case?
"action": {
          "description": "Describes the override action to be applied when rule matches.",
          "$ref": "#/definitions/ActionType"
        },

Reply:
Both are practically the same. I added the action in this way to match the existing "state" property. Do you think we should add the action property in the way AFD has?

Reply:
While they are the same, from user API, they kind of look little different. Since Rulestate is existing I thought changing it would break existing customers. But I don't think it would so I would rather we make it identical for state as well like AFD.

Reply:
Changed ActionType to match afd waf config